x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/index/IndexResolver.java

```diff
@@ -404,6 +404,10 @@ private static EsField createField(String fieldName, String typeName, Map<String
                 return new DateEsField(fieldName, props, isAggregateable);
             case UNSUPPORTED:
                 return new UnsupportedEsField(fieldName, typeName);
+            case GEO_SHAPE:
+                // TODO
+                // NORELEASE: clean this up
```

Comment:
This is here because SQL does not support GeoShape yet, stopgap until implemented. Tests fail, otherwise